### PR TITLE
Fixing messages page and updating README

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,5 @@ okta-oidc-tck
 samples-nodejs-express-4
 .DS_Store
 .vscode
-.samples.config.js
\ No newline at end of file
+.samples.config.js
+.samples.config.ts--
```
Comment (vijetmahabaleshwar-okta) [inline .gitignore]: This is a temporary backup file created on mac, when `sed` is used.

File: .gitignore [hunk 2]
```diff
@@ -7,4 +7,5 @@ okta-oidc-tck
 samples-nodejs-express-4
 .DS_Store
 .vscode
-.samples.config.js
\ No newline at end of file
+.samples.config.js
+.samples.config.js--
```
Comment (jmelberg-okta) [inline .gitignore]: I too had this file generated. Maybe we should look at the `setup.sh` script instead of attempting to ignore it?

Comment (jmelberg-okta) [reply .gitignore]: If you replace `sed -i --` with `sed -i ''` it will resolve this. See https://stackoverflow.com/a/7573438/7059401

Comment (vijetmahabaleshwar-okta) [reply .gitignore]: @jmelberg-okta - I have tried it but it works only on linux machines. On Macs, it threw an error if I remember correctly.

Comment (jmelberg-okta) [reply .gitignore]: Odd.. `sed -i ''` works great on my machine.

Image

Comment (vijetmahabaleshwar-okta) [reply .gitignore]: Yes, I tried it locally and it worked. Looks like it failed on travis. Let me try on travis and check.

**Comment by vijetmahabaleshwar-okta** [comment]
@jmelberg-okta - So, it was failing in travis as you can see in this run. Here's the error I get - `sed: can't read s/{yourOktaDomain}/[secure]/g: No such file or directory`
Looks like it only works on unix machines.

**Comment by jmelberg-okta** [comment]
@vijetmahabaleshwar-okta Gotcha. Lets just ignore that file then - makes it simpler. :)

File: README.md [hunk 3]
```diff
@@ -37,9 +37,9 @@ export OKTA_DOMAIN={yourOktaOrgDomain}
 
 **NOTE:** Use only the domain part of your org url while setting OKTA_DOMAIN environment.
 
-* E.g - If your org url is https://myorg.oktapreview.okta.com, your OKTA_DOMAIN should be myorg.oktapreview
+* E.g - If your org url is https://myorg.oktapreview.com, your OKTA_DOMAIN should be myorg.oktapreview
 
-After setting up the environment variables, you need to run a script to update the configuration
+After setting up the environment variables, you need to run a script to update the configuration & also clone the express resource server repo, which will be used to test `/messages` page in the sample application
```
Comment (jmelberg-okta) [inline README.md]: test `/messages` page -> test the `/messages` page

Comment (jmelberg-okta) [reply README.md]: & also clone -> and clone

Comment (jmelberg-okta) [reply README.md]: Don't forget the `.`! 🤓

Comment (vijetmahabaleshwar-okta) [reply README.md]: Thanks for the comments. Updated the PR

**Pull request merged**